### DISCUSSION TITLE
Choose entries per page in admin

### DISCUSSION THREAD

**Discussion by gixxy22** [craftcms/cms#9714] [Ideas] [answered]
The ability to select entries per page in admin would be useful, currently it defaults to 100, would be great to be able to select 10,25,50,100,250 etc

**Comment by brandonkelly** [comment, marked as answer]
The reason this isn’t currently possible is because there are already some cases where element actions will time out when being performed on 100 elements, and that would be much more frequent with anything larger. However we are going to start performing some element actions via the queue in Craft 4, which would make this change more viable.

**Reply by gixxy22** [reply]
thanks brandon, even smaller amounts would be nice to have, 25, 50, 100 as options.

cheers
